src/HTTPBackend.js

```diff
@@ -30,20 +30,46 @@ export default function(options) {
 
 	function parseResponse(request, response) {
 		return new Promise(function(resolve, reject) {
-			response.json().then((json) => {
-
-				let headers = { ...request.header };
+			let headers = { ...request.header };
+			delete headers.resource;
+			delete headers.parameters;
+			delete headers.queryParameters;
+
+			response.text().then(text => {
```

Comment:
I usually wrap `.text()` or `.json()` in an if statement like this:

```js
if (response.ok) {
  response.text()...
} else {
  reject("Server responded with status " + response.status);
}
```

Not sure if there's really much of a difference, but just thought I'd throw it out there.